readthedocs/core/logs.py

```diff
@@ -1,12 +1,11 @@
 import sys
 import warnings
-
 from io import StringIO
 
 import structlog
-from structlog.dev import _pad, plain_traceback
-
 from django_structlog.middlewares.request import RequestMiddleware
+from structlog.dev import _pad, plain_traceback
+from structlog.processors import _figure_out_exc_info
```

Comment:
We can just copy this method if we don't want to rely on this private function

https://github.com/hynek/structlog/blob/bcfc7f9e60640c150bffbdaeed6328e582f93d1e/src/structlog/processors.py#L519-L531

Reply:
I think it's fine for now.

Reply:
I found it to be fairly obscure and the private function didn't help to clarify what's going on. I think it'd be nice if @stsewd copies in the original function and adapts it to our needs with some comments on what it's doing :) But I think the Python2-specific parts can be removed.